src/microbe_stage/MicrobeHUD.cs

```diff
@@ -775,6 +775,11 @@ public void PauseButtonPressed()
         }
     }
 
+    public void SetupPressEditorTutorial(TutorialState tutorialState)
```

Comment:
```suggestion
    public void SendEditorButtonToTutorial(TutorialState tutorialState)
```
To have more consistent naming.